plan/prepare_test.go

```diff
@@ -0,0 +1,51 @@
+package plan_test
```

Comment:
Need a license.

Reply:
Will this test case cover this bug? If we revert the modification in common_plans.go, will this case fail?

Reply:
It will fail.